### ISSUE TITLE
Add new member

### ISSUE THREAD

**Issue by kuznetsovandrey76** [meerkats-sundial/sundial-frontend#12]
When I add myself to the team page and refresh the page all changes are gone

**Comment by ejuor** [comment]
Hello @kuznetsovandrey76! Unfortunately, our code is yet to implement state fetching from `localStorage` which would allow your changes to be saved even after a page refresh. It's a good feature to have so it it'll definitely come later. 

Thanks for your interest in our project.

**Comment by hoatran1403** [comment]
yep that feature gonna come soon with `localStorage` and database as well
